crates/autopilot/src/database/onchain_order_events.rs

```diff
@@ -235,59 +235,12 @@ impl<T: Sync + Send + Clone, W: Sync + Send + Clone> EventStoring<ContractEvent>
     }
 
     async fn append_events(&mut self, events: Vec<EthContractEvent<ContractEvent>>) -> Result<()> {
-        let order_placement_events = events
-            .clone()
-            .into_iter()
-            .filter(|EthContractEvent { data, .. }| {
-                matches!(data, ContractEvent::OrderPlacement(_))
-            })
-            .collect();
-        let invalidation_events = get_invalidation_events(events)?;
-        let invalided_order_uids = extract_invalidated_order_uids(invalidation_events)?;
-        let (custom_order_data, quotes, broadcasted_order_data, orders) = self
-            .extract_custom_and_general_order_data(order_placement_events)
-            .await?;
-
-        let _timer = DatabaseMetrics::get()
-            .database_queries
-            .with_label_values(&["append_onchain_order_events"])
-            .start_timer();
-        let mut transaction = self.db.pool.begin().await?;
-
```

Comment:
In the `replace_events` we also do:

- `delete_invalidations`
- `mark_as_reorged`

is that ok?

Reply:
Ah ok, that is what you meant with the range out of bounds. Clear.